submissions/antonkottans/hooli-style-popup/index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Popup window</title>
+</head>
+
+<body class="body">
+    <header>
+        <nav>
+            <ul class="nav-menu">
+                <li class="nav-menu-item">
+                    <a class="nav-menu-item-link" href="#">Mail</a>
+                </li>
+                <li class="nav-menu-item">
+                    <a class="nav-menu-item-link" href="#">Images</a>
+                </li>
+                <li class="nav-menu-item">
+                    <input type="checkbox" class="popup-checkbox hidden-checkbox" id="popup-checkbox">
+                    <label for="popup-checkbox" class="popup-checkbox-label">
+                        <img class="nav-image" src="img/popup-button.png" alt="popup-menu">
```

Comment:
Why do you leave the populated `alt` attribute for this icon if you remove it from the rest icons?

Reply:
i did it via replace all, three img tags had different parameters order, sorry